### ISSUE TITLE
E340 floodlight

### ISSUE THREAD

**Issue by Jaseore** [fuatakgun/eufy_security#1196]
Ever since I’ve set up live streams, the floodlights appear to stay on after a motion detection even though they are set to go off after 30seconds is this normal?

**Comment by Jaseore** [comment]
> @Jaseore did you ever figure this out?
> 
> I can't even get the E340 to stream at all. Only thing that works is the poster.

No, streaming was fine if you turn on rtsp in storage, however once motion was detected and the floodlight came on it wouldn’t switch off after the set time gave up in the end as it has to be issue with camera firmware I think

**Comment by IVIyg0t** [comment]
> > @Jaseore did you ever figure this out?
> > I can't even get the E340 to stream at all. Only thing that works is the poster.
> 
> No, streaming was fine if you turn on rtsp in storage, however once motion was detected and the floodlight came on it wouldn’t switch off after the set time gave up in the end as it has to be issue with camera firmware I think

Gotcha.  Yeah, I was hoping to avoid using RTSP... But maybe that's the only way to get it to work...
